src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Sort.txt

```diff
@@ -1,4 +1,24 @@
-﻿>> Sort([-2, -1, 5, 1, 2], Value)
+﻿#SETUP: OptionSetSortTestSetup
+
+>> Sort(t1, OptionSetField1, SortOrder.Ascending)
+Table({OptionSetField1:OptionSet.option_1,StrField1:"test1"},{OptionSetField1:OptionSet.option_2,StrField1:"test2"})
+
+>> Sort(t1, OptionSetField1, SortOrder.Descending)
+Table({OptionSetField1:OptionSet.option_2,StrField1:"test2"},{OptionSetField1:OptionSet.option_1,StrField1:"test1"})
+
+>> Sort(t2, OptionSetField1, SortOrder.Ascending)
+Table({OptionSetField1:OptionSet.option_1,StrField1:"test1"},{OptionSetField1:OptionSet.option_1,StrField1:"test3"},{OptionSetField1:OptionSet.option_2,StrField1:"test2"},{OptionSetField1:OptionSet.option_2,StrField1:"test4"})
+
+>> Sort(t2, OptionSetField1, SortOrder.Descending)
+Table({OptionSetField1:OptionSet.option_2,StrField1:"test2"},{OptionSetField1:OptionSet.option_2,StrField1:"test4"},{OptionSetField1:OptionSet.option_1,StrField1:"test1"},{OptionSetField1:OptionSet.option_1,StrField1:"test3"})
+
+>> Sort(t3, OptionSetField1, SortOrder.Ascending)
```

Comment:
@MikeStall  This test fails due to blank option set issue

Reply:
I have commented out the failing/unsupported test case for now

Reply:
This is hitting #731. 
But it's just happening in the test harness when trying to print the table  w/ option set. 
If we wrap in a Concat(), we avoid the test bug and still excercise the code. 

Can you adjust test case like this:
>> Concat(Sort(t1, OptionSetField1, SortOrder.Ascending), ThisRecord.StrField1 & ",")
"test1,test2,"

Then the output is a string, which the harness handles. 
I'd mention 731 in the test comments too.